include/superior_mysqlpp/extras/member_wrapper.hpp

```diff
@@ -0,0 +1,85 @@
+#pragma once
+
+namespace SuperiorMySqlpp
+{
+    namespace
```

Comment:
I wouldn't use anonymous namespace in header file. Please use `detail` namespace instead.